### PR TITLE
Fix output JSON structure for Foundry

### PR DESCRIPTION
**Pull request by tech-ticks**
- Rename and case-convert some attributes
- Add data consistency checks

### CODE REVIEW

File: scripts/generators/foundry.py [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
 import fire
-import time
 import datetime
 
-secrets = json.load(open('../../secrets.json'))
+# Moves with fields that are problematic for the export
+# - Non-moves such as "Run Away" are excluded
+# - Struggle and Growl have attributes like "Strength/special"
+# - Copycat Damage1 is "Same as the copied move"
+# - Simple Beam is rolled with Empathy. This skill exists neither on the Pokémon sheet nor on the trainer sheet (?)
```
Comment (Willowlark) [inline scripts/generators/foundry.py:12]: Empathy is Allure for Trainers, the trainer sheets don't have labeled correctly. it comes up alot, lol

File: scripts/generators/foundry.py [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
-import pandas as pd
-import re
 from shutil import copy
 import fire
-import time
 import datetime
 
-secrets = json.load(open('../../secrets.json'))
+# Moves with fields that are problematic for the export
+# - Non-moves such as "Run Away" are excluded
```
Comment (Willowlark) [inline scripts/generators/foundry.py:9]: These are moves available to all Pokemon, may want to have them included. What's causing them not to work?

File: scripts/generators/foundry.py [hunk 3]
```diff
@@ -1,20 +1,21 @@
...
+# - Struggle and Growl have attributes like "Strength/special"
```
Comment (Willowlark) [inline scripts/generators/foundry.py:10]: Those are in fact, interchangable. Not sure if Foundry can support the either other, but the bot and srd do via having two entries, one with each stat.

File: scripts/generators/foundry.py [hunk 4]
```diff
@@ -1,20 +1,21 @@
...
+# - Copycat Damage1 is "Same as the copied move"
```
Comment (Willowlark) [inline scripts/generators/foundry.py:11]: No idea how to make that work with Foundry, but it is correct

File: scripts/generators/foundry.py [hunk 5]
```diff
@@ -1,20 +1,21 @@
...
+# - Simple Beam is rolled with Empathy. This skill exists neither on the Pokémon sheet nor on the trainer sheet (?)
+# - Lovely Kiss Accuracy1 is "Missing beauty"
```
Comment (Willowlark) [inline scripts/generators/foundry.py:13]: Correct, it's 5-Beauty which a unique case.

**Review by Willowlark — APPROVED** [review]
Clarified the exception case moves for you

File: scripts/generators/foundry.py (hunk 1 above)
Comment (tech-ticks) [reply scripts/generators/foundry.py:12]: So the attribute roll should just be for Allure instead? I'm confused, even if it's mislabeled on the trainer sheet, there's still no Empathy on Pokémon

Comment (Willowlark) [reply scripts/generators/foundry.py:12]: Pokemon have Allure, Trainers have Empathy. Neither has both. (Empathy is understanding pokemon speak, which pokemon needless to say can do, and Allure is making Humans understand you, which humans can do already)

File: scripts/generators/foundry.py (hunk 2 above)
Comment (tech-ticks) [reply scripts/generators/foundry.py:9]: Nothing, but these common actions already have their own fields on the character sheet. It'd be confusing if they're  included as moves as well, which is why I'm sticking to "official" moves that not every Pokémon has

File: scripts/generators/foundry.py (hunk 3 above)
Comment (tech-ticks) [reply scripts/generators/foundry.py:10]: Yup, the split versions like "Growl (Cute)" etc. are included as separate entries, only the aggregate one is skipped

File: scripts/generators/foundry.py (hunk 2 above)
Comment (Willowlark) [reply scripts/generators/foundry.py:9]: Aaaaah, makes sense. Not a bad idea either.

File: scripts/generators/foundry.py (hunk 4 above)
Comment (tech-ticks) [reply scripts/generators/foundry.py:11]: By leaving the attribute empty and adding a note to Added Effects probably

File: scripts/generators/foundry.py (hunk 3 above)
Comment (Willowlark) [reply scripts/generators/foundry.py:10]: Cool, Didn't realize that from the note. 👍

**Review by Willowlark — APPROVED** [review]

**Pull request merged**